data/json/monsters/lab_security_drones.json

```diff
@@ -0,0 +1,564 @@
+[
+  {
+    "id": "green_maintenance_check",
+    "type": "SPELL",
+    "name": { "str": "Green maintenance Badge Trait" },
+    "description": "Adds trait for badge check",
+    "valid_targets": [ "hostile" ],
+    "flags": [ "MUTATE_TRAIT", "SILENT" ],
+    "message": "the drone scans for your ID badge.",
+    "min_damage": 10000,
+    "max_damage": 10000,
+    "min_range": 15,
+    "max_range": 15,
+    "max_level": 1,
+    "shape": "line",
+    "effect": "mutate",
+    "effect_str": "GREEN_MAINTENANCE_ALARM"
+  },
+  {
+    "type": "mutation",
+    "id": "GREEN_MAINTENANCE_ALARM",
+    "name": { "str": "valued employee" },
+    "points": 2,
+    "description": "A silent mutation to work with lab security drones and badges.",
+    "purifiable": false,
+    "player_display": false
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "bot_green_maintenance_badge_check",
+    "recurrence_min": 10,
+    "recurrence_max": 60,
+    "condition": { "and": [ { "not": { "u_has_item": "id_science_maintenance_green" } }, { "u_has_trait": "GREEN_MAINTENANCE_ALARM" } ] },
```

Comment:
So this only happens if you already have the GREEN_MAINTENANCE_ALARM trait? It does not make much sense to add the trait if you already have it.

Reply:
pulled these out and did some testing.  They appear to be redundant and not effecting the EOC chain so I've removed them.